WMF Framework/WMFExploreFeedContentController.m

```diff
@@ -448,7 +448,11 @@ - (NSDictionary *)exploreFeedPreferencesInManagedObjectContext:(NSManagedObjectC
     WMFKeyValue *keyValue = [moc wmf_keyValueForKey:WMFExploreFeedPreferencesKey];
     NSDictionary *exploreFeedPreferences = (NSDictionary *)keyValue.value;
     if (exploreFeedPreferences && [exploreFeedPreferences objectForKey:WMFExploreFeedPreferencesGlobalCardsKey]) {
-        return exploreFeedPreferences;
+        if ([self exploreFeedPreferencesNeedMigration:exploreFeedPreferences]) {
```

Comment:
I know you mention this is placeholder so no need to address it now, but for a manual-ish migration like this (i.e. nothing handled automatically by Core Data), we like to kick it off from `MWKDataStore.m`'s `performUpdatesFromLibraryVersion` method. We already have set up some migrations for our legacy event logging, so you could tack it on after the `migrateShareUsageAndInstallIDToUserDefaults` [line](https://github.com/wikimedia/wikipedia-ios/blob/b733d672810fda8a55e366919ca66011d433e57e/Wikipedia/Code/MWKDataStore.m#L442).

Reply:
Thanks Toni! I've copied this and added to my running notes for the migration piece.